src/feature/coffeeChat/ui/CoffeeChatListView.tsx

```diff
@@ -16,24 +19,38 @@ export const CoffeeChatListView = () => {
       <div>정보를 불러오는 중 문제가 발생하였습니다. 새로고침해주세요.</div>
     );
   }
+  if (
+    coffeeChatListData?.type === 'success' &&
+    Array.isArray(coffeeChatListData.data) &&
+    coffeeChatListData.data.length === 0
+  ) {
```

Comment:
Array.isArray 대신 이렇게 바꾸면 혹시 작동하나요? coffeeChatListData.type이 success면 무조건 데이터가 리스트인데 isArray로 체크하는 과정이 조금 어색한 거 같습니다.
```suggestion
  if (
    coffeeChatListData !== undefined &&
    coffeeChatListData.type === 'success' &&
    coffeeChatListData.data.length === 0
  ) {
```

Reply:
이것도     coffeeChatListData.data.coffeeChatList.length === 0 로 수정했습니다